### ISSUE TITLE
error when trying to rebuild stacks:

### ISSUE THREAD

**Issue by russelljjarvis** [scidash/docker-stacks#19]
When trying to rebuild neuronunit:
```
Installed /opt/conda/lib/python3.5/site-packages/pyNeuroML-0.3.0-py3.5.egg
  File "/opt/conda/lib/python3.5/site-packages/pyNeuroML-0.3.0-py3.5.egg/pyneuroml/neuron/__init__.py", line 64
    def export_to_neuroml1(hoc_file, nml1_file_name, level=1, validate=True):
SyntaxError: import * only allowed at module level

error: The 'pyneuroml==9999' distribution was not found and is required by neuronunit
```

**Comment by rgerkin** [comment]
@russelljjarvis 
I observed this as well, and I fixed this by switching to `pip install` in 78e5643c21bca62941d3a082c45642435225b2b6, which doesn't both to check the version number.  I need to use these high version numbers because it is the only way to force python to install the latest version from source, and for me to not always have to manually update that.  An alternative would be to harvest a version number from source and dynamically insert it as an ARG into Docker during the build, but that seems like too much of a hack.  

I am using `pip install --process-dependency-links -e`, which 1) processes the links to non-pypi sources, i.e. github and 3) installs as a developmental package, which means it installs it in place (where it was downloaded to), and so you can edit it in place at the source location to get new behavior without re-installing or adding it to the path (so I think the `ENV` lines at the end are now obsolete.  

Anyway, it builds for me now after this fix.

**Comment by russelljjarvis** [comment]
Okay trying this now.
